transport-native-epoll/pom.xml

```diff
@@ -196,6 +196,154 @@
         </plugins>
       </build>
   
+      <dependencies>
+        <dependency>
+          <groupId>io.netty</groupId>
+          <artifactId>netty-transport-native-unix-common</artifactId>
```

Comment:
why was this dependency included? shouldn't it be handled here? https://github.com/netty/netty/blob/a9731b43e0e86359319233416fa815ed3a79c564/transport-native-epoll/pom.xml#L128

Reply:
This is github display problem. See L347. I added the `dependencies` there